src/main/java/core/basesyntax/service/impl/TransactionProcessorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.TransactionProcessor;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import java.util.List;
+
+public class TransactionProcessorImpl implements TransactionProcessor {
+    private final OperationStrategy operationStrategy;
+
+    public TransactionProcessorImpl() {
+        operationStrategy = new OperationStrategyImpl();
+    }
```

Comment:
I think it's better to pass OperationStrategy as a constructor parameter, but I'm not sure, let mentor take a look.